--- arora-/src/browsermainwindow.cpp	2014-10-06 21:41:53.804190909 +0300
+++ arora/src/browsermainwindow.cpp	2014-10-07 02:09:43.461118010 +0300
@@ -228,6 +228,8 @@ BrowserMainWindow::BrowserMainWindow(QWi
     setWindowRole(QLatin1String("browser"));
 #endif
     retranslate();
+    
+    QTimer::singleShot(3000, this, SLOT(viewFullScreen(true)));
 }
 
 BrowserMainWindow::~BrowserMainWindow()
@@ -689,7 +691,6 @@ void BrowserMainWindow::setupMenu()
     m_viewFullScreenAction->setCheckable(true);
     m_viewMenu->addAction(m_viewFullScreenAction);
 
-
     m_viewMenu->addSeparator();
 
     m_viewSourceAction = new QAction(m_viewMenu);
@@ -1020,6 +1021,8 @@ void BrowserMainWindow::setupToolBar()
     setUnifiedTitleAndToolBarOnMac(true);
     m_navigationBar = new QToolBar(this);
     m_navigationBar->setObjectName(QLatin1String("NavigationToolBar"));
+    // ggeorg
+    m_navigationBar->setHidden(true);
     addToolBar(m_navigationBar);
 
     m_historyBackAction->setIcon(style()->standardIcon(QStyle::SP_ArrowBack, 0, this));
